docs/source/io.rst

```diff
@@ -17,6 +17,45 @@ Video
 .. autofunction:: write_video
 
 
+Low-level video API
+-------------------
+
+In addition to the :mod:`read_video` function, we provide a high-performance 
+low-level API for more fine-grained control compared to the old API.
+We expose it to users using TORCHBIND C++ registration, and allow users
+to access it via :mod:`torch.classes` import or via :mod:`torchvision.io`. For example
```

Comment:
This is an implementation detail, and I would only mention that it supports torchscript.

I would not mention that it is exposed via `torch.classes` neither, as it's confusing and, again, and implementation detail.